packages/meter/src/spectrum-meter.css

```diff
@@ -11,314 +11,361 @@ governing permissions and limitations under the License.
 */
 
 /* THIS FILE IS MACHINE GENERATED. DO NOT EDIT */
-:host([size='s']) {
-    --spectrum-progressbar-border-radius: var(
-        --spectrum-progressbar-s-border-radius
-    ); /* .spectrum-ProgressBar--sizeS */
-    --spectrum-progressbar-indeterminate-fill-width: var(
-        --spectrum-progressbar-s-indeterminate-fill-width,
-        var(--spectrum-global-dimension-static-percent-70)
+:host {
+    --spectrum-progressbar-animation-ease-in-out-indeterminate: cubic-bezier(
+        0.45,
+        0,
+        0.4,
+        1
+    ); /* .spectrum-ProgressBar */
+    --spectrum-progressbar-animation-duration-indeterminate: 1000ms;
+    --spectrum-progressbar-corner-radius: var(--spectrum-corner-radius-100);
+    --spectrum-progressbar-fill-size-indeterminate: 70%;
+    --spectrum-progressbar-size-2400: 192px;
+    --spectrum-progressbar-size-2500: 200px;
+    --spectrum-progressbar-size-2800: 224px;
```

Comment:
@pfulton questions about these static values...should they be static instead of system tokens?

In particular, the `--spectrum-progressbar-animation-duration-indeterminate` existing outside of a token makes it difficult to customize for `preferes-reduced-motion`. This _also_ happens to be the approach we use at VRT time to prevent animation from failing the regression. Thoughts?

Reply:
I may be misunderstanding, but are you suggesting that these not use custom properties? Or are you suggesting that they be defined elsewhere (and/or differently)?

Happy to make whatever changes would make life easier for folks to work with the component.

Reply:
Having static values in the component is surprising, but I may also have missed it in other packages for them not having adverse side effects on the component.

In particular, `--spectrum-progressbar-animation-duration-indeterminate: 1000ms;` prevents the system from controlling motion durations. It would be better if this used a value outlined [here](https://unpkg.com/browse/@spectrum-css/tokens@4.0.0/dist/css/spectrum/custom-vars.css#L19) or [here](https://unpkg.com/browse/@spectrum-css/vars@8.0.0/css/globals/spectrum-animationGlobals.css) so that we could universally alter them.

The specific use case that we support this way today is our VRT coverage, moving things botches screenshots. However, this should also be the basis of [prefers reduced motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) for Spectrum patterns, as well.

Reply:
@yosevu @pfulton have we made any progress on this? I'm gonna move this PR back into draft for now, but happy to look more closely when we've got an update ready.

Reply:
@Westbrook Sorry, this fell off the radar with the run up to the holiday break. We're going to move these animation custom props out of this component and up into the `@spectrum-css/tokens` package. Then, we'll push a new version of the progress bar with that update. I'll update this PR with those changes once they're available.

Reply:
@Westbrook ok, would you mind taking a peek at the changes [here](https://github.com/adobe/spectrum-css/pull/1572)? I've created a [handful of animation vars](https://github.com/adobe/spectrum-css/blob/cab7fb9afe5f5249ee9ab305183a06a242afd324/components/tokens/custom-spectrum/custom-vars.css#L19-L36) in the tokens package, and Progress Bar is now relying on those "globals" instead of creating them directly from within the component.

Reply:
This looks good to me! If we can get that shipped then I think this pattern will be good to go 🤞🏼

Reply:
@Westbrook ok, that's been merged and released. It's available in: `@spectrum-css/progressbar@3.0.1`